modules/nf-core/deeptmhmm/main.nf

```diff
@@ -0,0 +1,62 @@
+process DEEPTMHMM {
+    tag '$deeptmhmm'
+    label 'process_medium'
+
+    conda "bioconda::pybiolib=1.1.1367"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/pybiolib:1.1.1367--pyhdfd78af_0':
+        'biocontainers/pybiolib:1.1.1367--pyhdfd78af_0' }"
+
+    input:
+    path fasta
```

Comment:
Can this please take a meta map. We require this in general now for any fasta channel, to make everything work better for pipelines that involve multiple genomes.

Reply:
And then also emit the meta map on the output channels too.

Reply:
Done, but messed Git History. So closing this pull request and opening new one...